Sharpmake.Generators/VisualStudio/Csproj.cs

```diff
@@ -337,7 +337,7 @@ internal class Content : ItemGroupItem, IResolvable
             {
                 public string Generator;
                 public string LastGenOutput;
-                public string CopyToOutputDirectory;
+                public ItemCopyType CopyToOutputDirectory = ItemCopyType.None;
```

Comment:
We have a very similar change in a changelist I'm about to submit a pull requst for, although it's called CopyToOutputDirectory. I prefer the name ItemCopyType though since it's more general.

@belkiss thoughts?